flexmeasures/ui/tests/utils.py

```diff
@@ -48,6 +52,42 @@ def mock_asset_response(
     return asset
 
 
+def mock_asset_response_with_kpis(
+    db,
+    asset_id: int = 1,
+    account_id: int = 1,
+    as_list: bool = True,
+    multiple: bool = False,
+) -> dict | list[dict]:
+    asset = mock_asset_response(
+        asset_id=asset_id, account_id=account_id, as_list=as_list, multiple=multiple
+    )
+    sensor: Sensor = db.session.execute(
+        select(Sensor).filter_by(event_resolution="PT24H")
+    ).scalar_one_or_none()
+
+    if not sensor:
+        # Create a sensor first with PT24H
+        sensor = Sensor(
+            name="KPI sensor",
+            generic_asset_id=asset["id"],
+            event_resolution="PT24H",
+            unit="kWh",
+            attributes={},
+        )
+        db.session.add(sensor)
+        db.session.flush()
+    sensors_to_show_as_kpis = [
+        {
+            "title": "My KPIs",
+            "sensor": sensor.id,
+            "default_function": "sum",
```

Comment:
@nhoening Why do we still mention the word "default" here, rather than just "function"?

Reply:
Not sure. I probably thought we wanted to keep it open for enhancements in the future, where multiple functions are possible, like in the simulations.

I believe it might be a while until that happens here, if at all, so making this simpler would be nice, actually.